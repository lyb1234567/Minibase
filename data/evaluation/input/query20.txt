Q(x,y,SUM(8*12)) :- S(x,y,z), 'a'<='aa'