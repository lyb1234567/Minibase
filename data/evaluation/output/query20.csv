1, 'smith', 96
2, 'anka', 96
4, 'rhcp', 96
5, 'bowie', 96
8, 'pistols', 96
