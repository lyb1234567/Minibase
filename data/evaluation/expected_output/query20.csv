1, 'smith', 96
8, 'pistols', 96
4, 'rhcp', 96
5, 'bowie', 96
2, 'anka', 96